grepdemos/cal.com
# perf: Added prisma optimize support with flag

## What does this PR do?



- Fixes #15795 (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

https://www.loom.com/share/30ffb9c4cd5843a08c0ec0e5a35fdef6?sid=e428ac8c-8799-442a-bd31-f84cf410c102

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Are there environment variables that should be set?
Yes! As requested in my previous PR. env variable should  have ENABLE_OPTIMIZE_EXTENSION="true" to run prisma optimize. Else Prisma Optimize won't start.
